src/components/settings/EditOrganizationInformationsDialog.tsx

```diff
@@ -0,0 +1,323 @@
+    },
+    validationSchema: object().shape({
+      email: string().email('text_620bc4d4269a55014d493fc3'),
+    }),
+    enableReinitialize: true,
+    validateOnMount: true,
+    onSubmit: async ({ ...values }, formikBag) => {
+      await updateOrganizationInformations({
+        variables: {
+          input: {
+            logo,
+            ...values,
+          },
+        },
+        refetchQueries: ['getOrganizationInformations'],
+      })
+      ;(ref as unknown as RefObject<DialogRef>)?.current?.closeDialog()
+      if (mounted.current) {
+        formikBag.resetForm()
+      }
+    },
+  })
+
+  const getBase64 = (file: Blob) => {
+    var reader = new FileReader()
+
+    if (file.size > FILE_MAX_SIZE) {
+      setLogoUploadError(true)
+      return
+    }
+
+    reader.readAsDataURL(file)
+    reader.onload = function () {
+      setLogo(reader?.result?.toString())
+    }
+    reader.onerror = function (error) {
+      // eslint-disable-next-line no-console
+      console.error('Error: ', error)
+    }
+  }
+
+  useEffect(() => {
+    mounted.current = true
+
+    return () => {
+      mounted.current = false
+    }
+  }, [])
+
+  return (
+    <Dialog
+      ref={ref}
+      title={translate('text_62728ff857d47b013204c76c')}
+      description={translate('text_62728ff857d47b013204c77a')}
+      actions={({ closeDialog }) => (
+        <>
+          <Button
+            variant="quaternary"
+            onClick={() => {
+              closeDialog()
+              if (mounted.current) {
+                formikProps.resetForm()
+              }
+            }}
+          >
+            {translate('text_62728ff857d47b013204c7e4')}
+          </Button>
+          <Button
+            variant="primary"
+            disabled={!formikProps.isValid}
+            onClick={async () => await formikProps.handleSubmit()}
+          >
+            {translate('text_62ab2d0396dd6b0361614db2')}
+          </Button>
+        </>
+      )}
+    >
+      <Content>
+        <FormSection>
+          <AvatarContainer>
+            {logo || organization?.logoUrl ? (
+              <Avatar size="large" variant="connector">
+                <img
+                  src={(logo || organization?.logoUrl) as string}
+                  alt={`${organization?.name}'s logo`}
+                />
+              </Avatar>
+            ) : (
+              <Avatar
+                size="large"
+                variant="company"
+                identifier={organization?.name || ''}
+                initials={(organization?.name || '')
+                  .split(' ')
+                  .reduce((acc, n) => (acc = acc + n[0]), '')}
+              />
+            )}
+            <AvatarUploadWrapper>
+              <Button variant="secondary" onClick={() => hiddenFileInputRef?.current?.click()}>
+                {translate('text_62ab2d0396dd6b0361614d18')}
+              </Button>
+              <Typography variant="caption" color={logoUploadError ? 'red' : ''}>
```

Comment:
Alright so, sorry not your fault, I need to type this.
But you can't use generic colors like that to stay in the design system.
The red you used is not the one from the spec also, it should be `#DC3309` 
So to check which one it is, go to the `colorPalette` file, check the value (here `#DC3309` --> error.main` and then use : 

```tsx
<Typography variant="caption" color={!!logoUploadError && 'error.main'}>
```

Reply:
FTR, it's now `danger600` and not `error.main` after this [MR](https://github.com/getlago/lago-front/pull/195) have been merged